[2025-03-21T19:03:55.423+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T19:03:55.462+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather-two-source.create_insert_dim_tables manual__2025-03-21T19:03:29.871520+00:00 [queued]>
[2025-03-21T19:03:55.480+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather-two-source.create_insert_dim_tables manual__2025-03-21T19:03:29.871520+00:00 [queued]>
[2025-03-21T19:03:55.481+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-21T19:03:55.503+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_insert_dim_tables> on 2025-03-21 19:03:29.871520+00:00
[2025-03-21T19:03:55.514+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1437) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-21T19:03:55.518+0000] {standard_task_runner.py:72} INFO - Started process 1442 to run task
[2025-03-21T19:03:55.518+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather-two-source', 'create_insert_dim_tables', 'manual__2025-03-21T19:03:29.871520+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/weather-two-source.py', '--cfg-path', '/tmp/tmpz0or4ycu']
[2025-03-21T19:03:55.523+0000] {standard_task_runner.py:105} INFO - Job 137: Subtask create_insert_dim_tables
[2025-03-21T19:03:56.043+0000] {task_command.py:467} INFO - Running <TaskInstance: weather-two-source.create_insert_dim_tables manual__2025-03-21T19:03:29.871520+00:00 [running]> on host c25e9da6226b
[2025-03-21T19:03:56.198+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather-two-source' AIRFLOW_CTX_TASK_ID='create_insert_dim_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T19:03:29.871520+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T19:03:29.871520+00:00'
[2025-03-21T19:03:56.201+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T19:03:56.251+0000] {base.py:84} INFO - Retrieving connection 'rizqi-neon'
[2025-03-21T19:03:56.626+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T19:03:56.644+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T19:03:56.645+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather-two-source, task_id=create_insert_dim_tables, run_id=manual__2025-03-21T19:03:29.871520+00:00, execution_date=20250321T190329, start_date=20250321T190355, end_date=20250321T190356
[2025-03-21T19:03:56.744+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T19:03:56.760+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
